doc/subsystems/test/sanitycheck.rst

```diff
@@ -385,6 +385,64 @@ extra_sections: <list of extra binary sections>
     extra, unexpected sections in the Zephyr binary unless they are named
     here. They will not be included in the size calculation.
 
+harness: <list of harness>
```

Comment:
plural of harness is harnesses, so this should be ``list of harnesses``

Reply:
Changed as suggested by @nashif.